src/visualization/types/Graph/view.tsx

```diff
@@ -270,7 +270,8 @@ const XYPlot: FC<Props> = ({
       }
     })
 
-    if (inAnnotationWriteMode) {
+    // only pass interaction handler to giraffe if annotations write mode is enabled and cellID is present
```

Comment:
i think the comment would be more useful if it explained why cell id needs to be present. it's pretty clear why `inAnnotationWriteMode` needs to be `true`, and it's clear what it means when it's true. it's not as clear what it means for the cell id to not exist